### PR TITLE
Ensures active_tab is in context of all views

### PR DESCRIPTION
**Pull request by whitej6**
# Closes: #4986 
# What's Changed

Initial pass on ensuring active_tab is set for all views that overload `get_extra_context`

# TODO

- [ ] Explanation of Change(s)
- [ ] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [ ] Attached Screenshots, Payload Example
- [ ] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Example Plugin Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

File: nautobot/core/views/mixins.py [hunk 1]
```diff
@@ -483,7 +483,9 @@ def get_extra_context(self, request, instance=None):
         request: The current request
         instance: The object being viewed
         """
-        return {}
+        return {
+            "active_tab": request.GET.get("tab", "main"),
```
[the remark this answers is not in the document]
Comment (whitej6) [reply nautobot/core/views/mixins.py:487]: There's a conditional somewhere else in the file that is just for detail view that it calls the method. I thought the same thing.

[the remark this answers is not in the document]
Comment (whitej6) [reply nautobot/core/views/mixins.py:487]: Well not the same file, and please correct me if my interpretation of this conditional https://github.com/nautobot/nautobot/blob/181326820e0b8298c2573831ba2170954ede7708/nautobot/core/views/renderers.py#L232

[the remark this answers is not in the document]
Comment (glennmatthews) [reply nautobot/core/views/mixins.py:487]: Looks like it's also done in the `else` case here: https://github.com/nautobot/nautobot/blob/181326820e0b8298c2573831ba2170954ede7708/nautobot/core/views/renderers.py#L264

[the remark this answers is not in the document]
Comment (glennmatthews) [reply nautobot/core/views/mixins.py:487]: So maybe could key off of `if instance is not None`?

**Comment by joewesch** [comment]
Does this handle hardcoded items like this?
https://github.com/nautobot/nautobot/blob/15d9d9f04f8a420b8c6cf179955ad098bf2b61dc/nautobot/dcim/views.py#L1155

**Comment by whitej6** [comment]
> Does this handle hardcoded items like this?
> 
> https://github.com/nautobot/nautobot/blob/15d9d9f04f8a420b8c6cf179955ad098bf2b61dc/nautobot/dcim/views.py#L1155

Yes, the super was only added when active tab was not provided

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**